src/common/scrub_types.cc

```diff
@@ -133,12 +133,10 @@ inconsistent_obj_wrapper::set_auth_missing(const hobject_t& hoid,
 {
   for (auto pg_map : maps) {
     auto oid_object = pg_map.second->objects.find(hoid);
+    // XXX: Should we just continue if object not found?
```

Comment:
there is chance that neither does the map have the object. for example, assuming we have a active set of osd.0, osd.1 and osd.2. object `foo` could be missing in osd.1, so, in `PG::build_scrub_map_chunk()`, pg just skips it when preparing the scrub map. but neither osd.0 or osd.2 can provide an authorized copy of `foo`.

so we cannot assert in this case.

and, i think, we should at least inform user that 

1. a file is missing on osd.1. if user knows the active set of that PG interval, we can ignore this.
1. the inconsistencies found on other osds. because we don't update the scrub store in `PGBackend::be_select_auth_object()`. yeah, we are good.

in other words, if user knows its a problem that an object is not found and he/she knows which shard/replica is missing, then we can just continue, otherwise, we cannot continue. we don't provide the active set info in the list-inconsistent-object output yet. but, yes, user can leverage other ceph command for this info.